src/systems/interactions.js

```diff
@@ -5,28 +5,16 @@ import { waitForDOMContentLoaded } from "../utils/async-utils";
 import { canMove } from "../utils/permissions-utils";
 
 function findHandCollisionTargetForHand(body) {
-  const driver = AFRAME.scenes[0].systems.physics.driver;
-  const numManifolds = driver.dispatcher.getNumManifolds();
-  const handPtr = Ammo.getPointer(body);
-  for (let i = 0; i < numManifolds; i++) {
-    const persistentManifold = driver.dispatcher.getManifoldByIndexInternal(i);
-    const body0ptr = Ammo.getPointer(persistentManifold.getBody0());
-    const body1ptr = Ammo.getPointer(persistentManifold.getBody1());
-    if (handPtr !== body0ptr && handPtr !== body1ptr) {
-      continue;
-    }
-    const numContacts = persistentManifold.getNumContacts();
-    for (let j = 0; j < numContacts; j++) {
-      const manifoldPoint = persistentManifold.getContactPoint(j);
-      if (manifoldPoint.getDistance() <= 10e-6) {
-        const object3D = driver.els.get(handPtr === body0ptr ? body1ptr : body0ptr).object3D;
-        if (object3D.el && object3D.el.components.tags && object3D.el.components.tags.data.isHandCollisionTarget) {
-          return object3D.el;
-        }
-        return null;
-      }
+  const world = this.el.sceneEl.systems["hubs-systems"].physicsSystem.world;
+  const handPtr = Ammo.getPointer(body.physicsBody);
+
+  if (world.collisions.has(handPtr) && world.collisions.get(handPtr).length > 0) {
+    const object3D = world.object3Ds.get(world.collisions.get(handPtr)[0]);
```

Comment:
Is this code correct? In the old version, if the hand collides with multiple things and the first one examined isn't a collision target, it continues to examine the other things. In this version, we give up.